intellect-newgen-style.csl

```diff
@@ -0,0 +1,560 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" page-range-format="minimal" demote-non-dropping-particle="never" default-locale="en-GB">
+  <!-- This style was edited with the Visual CSL Editor (https://editor.citationstyles.org/visualEditor/) -->
+  <info>
+    <title>Intellect Newgen Style</title>
+    <id>http://www.zotero.org/styles/intellect-newgen-style</id>
+    <link href="http://www.zotero.org/styles/intellect-newgen-style" rel="self"/>
+    <category citation-format="author-date"/>
+    <category field="social_science"/>
+    <summary>Style for Intellect Newgen Style</summary>
+    <updated>2020-08-19T09:16:06+00:00</updated>
+    <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
+  </info>
+  <macro name="conditionalusetitle">
+    <choose>
+      <if type="article-newspaper article-magazine" match="any">
+        <text variable="container-title"/>
+      </if>
+      <else>
+        <text variable="title" form="short"/>
+      </else>
+    </choose>
+  </macro>
+  <macro name="year">
+    <choose>
+      <if variable="original-date">
+        <date variable="original-date">
+          <date-part name="year"/>
+        </date>
+      </if>
```

Comment:
Should the publication year (`issued`) also be included when `original-date` is present?